src/Streamly/Internal/Data/Stream/IsStream/Type.hs

```diff
@@ -728,8 +711,8 @@ concatMapWith par f xs = bindWith par xs f
 -- Equivalent to:
 --
 -- @
--- concatMapFoldableWith f g = Prelude.foldr (f . g) S.nil
--- concatMapFoldableWith f g xs = S.concatMapWith f g (S.fromFoldable xs)
+-- concatMapFoldableWith f g = Prelude.foldr (f . g) D.nil
+-- concatMapFoldableWith f g xs = D.concatMapWith f g (D.fromFoldable xs)
```

Comment:
User examples should not mention D or K. We should always use `Stream` or `S` in the user documentation.